src/test/java/com/smartbear/readyapi/client/PluginTestStepBuilderTest.java

```diff
@@ -0,0 +1,34 @@
+package com.smartbear.readyapi.client;
+
+import com.smartbear.readyapi.client.model.PluginTestStep;
+import org.junit.Test;
+
+import static com.smartbear.readyapi.client.TestRecipeBuilder.newTestRecipe;
+import static com.smartbear.readyapi.client.teststeps.TestSteps.pluginTestStep;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class PluginTestStepBuilderTest {
+    @Test
+    public void buildsRecipeWithPluginTestStep() throws Exception {
+        TestRecipe testRecipe = newTestRecipe()
+                .addStep(pluginTestStep("MQTTPublishTestStep")
```

Comment:
Yes, we could make it generic - but do we want to have the configuration also for all test steps ? Also, for other test steps we have convinience methods so that user doesn't have to provide the test step type and I thought it's better to keep this explicit that you have to provide type only if it is a plugin provided test step. But I can change if you insist.